plugins/modules/vmware_cluster_vsan.py

```diff
@@ -20,9 +20,11 @@
 author:
 - Joseph Callen (@jcpowermac)
 - Abhijeet Kasurde (@Akasurde)
+- Mario Lenz (@mariolenz)
 requirements:
-    - Tested on ESXi 5.5 and 6.5.
+    - Tested on ESXi 6.7.
```

Comment:
@mariolenz Does this mean these new changes require ESXi 6.7?

Reply:
According to the documentation, ReconfigureComputeResource_Task is the replacement of ReconfigureComputeResource_Task since vSAN 6.2 which I think corresponds to ESXi 6.0U2. So it _should_ work          for vSAN 6.2 / ESXi 6.0U2 and later but I've _tested_ this with 6.7. U3, to be precise.